src/probe.c

```diff
@@ -35,6 +35,9 @@
 #include "probe.pio.h"
 #include "tusb.h"
 
+#define PADS_BANK0_GPIO12       _u(12)
```

Comment:
Feels like this should probably use the `#define` values from e.g. https://github.com/raspberrypi/debugprobe/blob/master/include/board_debug_probe_config.h and https://github.com/raspberrypi/debugprobe/blob/master/include/board_pico_config.h rather than using hard-coded pin numbers?

Reply:
You would be correct, I overlooked that. I thought all the defines I needed that existed were at [https://github.com/raspberrypi/pico-sdk/blob/6a7db34ff63345a7badec79ebea3aaef1712f374/src/rp2040/hardware_regs/include/hardware/regs/pads_bank0.h](url). My mistake